pkg/apis/componentconfig/types.go

```diff
@@ -610,6 +610,16 @@ type KubeControllerManagerConfiguration struct {
 	// concurrentGCSyncs is the number of garbage collector workers that are
 	// allowed to sync concurrently.
 	ConcurrentGCSyncs int32 `json:"concurrentGCSyncs"`
+	// nodeEvictionRate is the number of Node evictions NodeController does in ordinary circumstances
```

Comment:
nodeEvictionRate is the number of nodes per second on which pods are deleted in case of node failure when a zone is healthy